### PR TITLE
confirmation page content changes

### PR DESCRIPTION
**Pull request by mikail-khan**

### CODE REVIEW

File: app/views/ConfirmationView.scala.html [hunk 1]
```diff
@@ -31,13 +33,19 @@
         breakdownHelper: ConfirmationBreakdownHelper
 )
 
-@(grant: Grant, regularPay: BigDecimal, version: String)(implicit request: Request[_], messages: Messages)
+@(grant: Grant, calculatorVersion: String)(implicit request: Request[_], messages: Messages)
+
+    @now = @{
+        val formatter = DateTimeFormatter.ofPattern("dd MMMM yyyy")
+        LocalDate.now().format(formatter).toString
```
Comment (sureshhmrc) [inline app/views/ConfirmationView.scala.html:40]: you can use ViewUtils.datetoString(...)

File: app/views/ConfirmationView.scala.html [hunk 2]
```diff
@@ -91,5 +94,9 @@
         @p(Html(Messages(breakdownHelper.eligibility(gfp))))
     }
 
+    @p(Html(Messages("confirmation.p5")))
+
+    @p(Html(link(routes.FeedbackSurveyController.startSurvey(), "confirmation.print.label", true)))
```
Comment (sureshhmrc) [inline app/views/ConfirmationView.scala.html:99]: i guess this link is wrong for the print

File: conf/messages.en [hunk 3]
```diff
@@ -303,29 +303,45 @@ actualHours.error.required = Enter your actual hours
...
+confirmation.breakdown.reference-pay.p1.l2 = Divide by the {0} usual hours worked.
+confirmation.breakdown.reference-pay.p1.l3 =  Multiply by {0} hours worked.
+confirmation.breakdown.reference-pay.p2 = Pay for hours worked = {0}
+
+confirmation.breakdown.p3 = Then:
+confirmation.breakdown.reference-pay.p1 = Take the reference pay for the pay period:
+confirmation.breakdown.reference-pay.p1.l1 = Start with: {0}
+confirmation.breakdown.reference-pay.p1.l2 = Divide by the {0} usual hours worked.
+confirmation.breakdown.reference-pay.p1.l3 =  Multiply by {0} hours worked.
+confirmation.breakdown.reference-pay.p2 = Pay for hours worked = {0}
```
Comment (sureshhmrc) [inline conf/messages.en:340]: keys/messages duplicated, i guess

File: app/views/ConfirmationView.scala.html (hunk 1 above)
Comment (mikail-khan) [reply app/views/ConfirmationView.scala.html:40]: yeah eventually did - that needs to be removed.

File: app/views/ConfirmationView.scala.html (hunk 2 above)
Comment (mikail-khan) [reply app/views/ConfirmationView.scala.html:99]: yeah dummy links for now

File: app/views/ConfirmationView.scala.html [hunk 4]
```diff
@@ -91,5 +94,9 @@
...
+    @p(Html(link(routes.FeedbackSurveyController.startSurvey(), "confirmation.webchat.label", true)))
```
Comment (sureshhmrc) [inline app/views/ConfirmationView.scala.html:100]: same as above

**Pull request merged**